reg/udral/README.md

```diff
@@ -236,6 +236,27 @@ It is not necessary to include the leading and trailing name component separator
 Registers that are not known to the auto-configuration authority are to be silently ignored,
 which permits addition of implementation-defined ports without breaking compatibility with standard services.
 
+
+#### Type signature register
+
+In many use cases it is desirable to have the ability to statically check the compatibility between sender and 
+receiver of a message. This capability could be used both during automatic configuration and by manual configurator tools.
+The design allows this by adding an extra register to every service pub/sub group that contains a string _message type signature_.
+
+    uavcan.sub.main_drive.setpoint.type_sig
+
+Example signature would look like `(u12{u8}a5[f16]=i8)` , and be interpreted as
```

Comment:
```suggestion
Example signature would look like `(u12{u8}a5[f16]=i8)`, and be interpreted as
```

A slightly more formal specification would be nice.

Reply:
Sure, I'm just reluctant to spend more time than strictly necessary right now until we see some traction.